module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AuthorSearchTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Mink author search test class.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+namespace VuFindTest\Mink;
+
+/**
+ * Mink author search test class.
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ * @retry    4
+ */
+class AuthorSearchTest extends \VuFindTest\Integration\MinkTestCase
+{
+    /**
+     * Test searching for a known corporate author
+     *
+     * @return void
+     */
+    public function testCorporateAuthorSearch(): void
+    {
+        $page = $this->performSearch('corporate', 'Author');
+        $facets = $this->findCss($page, '#side-collapse-building a')->getText();
+        // We'll check for a known count from a known MARC file to confirm that
+        // results came back.
+        $this->assertTrue(strpos($facets, "author_relators.mrc 10") !== false);
```

Comment:
Use [assertStringContainsString](https://phpunit.readthedocs.io/en/9.6/assertions.html#assertstringcontainsstring) in both tests.

Reply:
Good idea -- fix pushed up. I'm going to run the whole test suite one last time, then merge if nothing fails.